eclair-core/src/main/scala/fr/acinq/eclair/transactions/Transactions.scala

```diff
@@ -190,6 +217,11 @@ object Transactions {
   }
   object RedeemInfo {
     sealed trait SegwitV0 extends RedeemInfo { def redeemScript: ByteVector }
+    /** @param publicKey the public key for this p2wpkh input. */
+    case class P2wpkh(publicKey: PublicKey) extends SegwitV0 {
+      override val redeemScript: ByteVector = Script.write(Script.pay2pkh(publicKey))
```

Comment:
A comment would be welcome here, as it's not obvious at all for a script newbie like myself.

Reply:
Agreed, we have that comment inside `bitcoin-kmp`, will add it here.

Reply:
Done in https://github.com/ACINQ/eclair/pull/3192/commits/527991a4f7d968c92f0739b82169a2adf05e6a18